proposals/lip-0051.md

```diff
@@ -1936,7 +1947,7 @@ def lock(
 
     availableBalance(address, tokenID) -= amount
     lockedAmount(address, module, tokenID) += amount
-    emitPersistentEvent(
+    emitEvent(
```

Comment:
mh, this is weird, are we sure there was no reason for this event to be persistent? Makes sense to me that it's not, but looks strange.

Reply:
I think it was just overlooked before. If for some reason a failure occurs and the state changes are reverted, we don't want this event to be emitted..

Reply:
ok, can be resolved from my side